HWIMO-BUILD

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright 2015, 2016, EMC, Inc.
+
+# debian packages expected...
+# nodejs >= 6.0.0
+# Input (environment variables):
+# * PKG_VERSION: The version of debian package, 
+#                such as: 1.3.1 (official release);
+#
+
+set -e
+set -x
+
+current_node_version=`node -v`
+. ~/.nvm/nvm.sh && nvm use 6
```

Comment:
if the version 6 is not yet installed, what is the behavior of the shell script?